src/server/req_stat.c

```diff
@@ -818,25 +818,16 @@ req_stat_sched(struct batch_request *preq)
 	preply->brp_choice = BATCH_REPLY_CHOICE_Status;
 	CLEAR_HEAD(preply->brp_un.brp_status);
 
-	psched = NULL;
-	if(strlen(preq->rq_ind.rq_status.rq_id) != 0) {
-		psched = find_scheduler(preq->rq_ind.rq_status.rq_id);
-		if(psched) {
-			rc = status_sched(psched, preq, &preply->brp_un.brp_status);
-		} else {
-			req_reject(PBSE_UNKSCHED, 0, preq);
-			return;
-		}
-	} else {
-		psched = (pbs_sched *) GET_NEXT(svr_allscheds);
-		while (psched != (pbs_sched *) 0) {
-			rc = status_sched(psched, preq, &preply->brp_un.brp_status);
-			if (rc != 0) {
-				break;
-			}
-			psched = (pbs_sched *) GET_NEXT(psched->sc_link);
+	for(psched = (pbs_sched *) GET_NEXT(svr_allscheds);
+			(psched != (pbs_sched *) 0);
```

Comment:
Don't use (pbs_sched *) 0, use NULL

Reply:
>Don't use (pbs_sched *) 0, use NULL

OK Changed it to NULL.  But if you see carefully this was existing code which was rearranged.